digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/queries.scala

```diff
@@ -182,6 +182,16 @@ object Queries {
     """
   }
 
+  def insertNumberProperty(assetId: Long, propertyId: Long, value: Option[Double]) = {
+    sqlu"""
+      insert into number_property_value(id, property_id, asset_id, value)
+      values (primary_key_seq.nextval, $propertyId, $assetId, $value)
+    """
+  }
+
+  def updateNumberProperty(assetId: Long, propertyId: Long, value: Option[Double]) =
+    sqlu"update number_property_value set value = $value where asset_id = $assetId and property_id = $propertyId"
+
   def updateNumberProperty(assetId: Long, propertyId: Long, value: Double) =
     sqlu"update number_property_value set value = $value where asset_id = $assetId and property_id = $propertyId"
 
```

Comment:
It is beginning to look like duplicates for me, can we try to adapt this for example the option could be put on the existing value:Double

Reply:
Fixed

Reply:
This will be improved in task DORTH-2394